impeller/core/vertex_buffer.h

```diff
@@ -5,14 +5,17 @@
 #ifndef FLUTTER_IMPELLER_CORE_VERTEX_BUFFER_H_
 #define FLUTTER_IMPELLER_CORE_VERTEX_BUFFER_H_
 
-#include <variant>
+#include <array>
 #include "impeller/core/buffer_view.h"
 #include "impeller/core/formats.h"
 
 namespace impeller {
 
+constexpr size_t kMaxVertexBuffers = 16;
+
 struct VertexBuffer {
-  std::variant<BufferView, std::vector<BufferView>> vertex_buffers;
+  std::array<BufferView, kMaxVertexBuffers> vertex_buffers;
```

Comment:
I think the VertexBuffer struct can keep the single buffer view slot, right?

Reply:
GLES relies on this struct in the command for the list of bindings. But I could potentially remove the usage of this from `impeller::Command` and put the properties inline.

Reply:
I'll try to make this change now so we can avoid changing the VertexBuffer struct.

Reply:
Alright, I've changed this so that `impeller::Command` no longer stores a `VertexBuffer` and changed `VertexBuffer` back to its original state.